server/render-template/index.tsx

```diff
@@ -43,7 +43,7 @@ function renderTemplate(
   templateType: ServiceNames,
   templateData: Record<string, unknown>
 ) {
-  const Node: React.ComponentType<any> = templateMapper[templateType];
+  const Node: React.ComponentType<any> = templateMapper[templateType]; // eslint_disable_line
```

Comment:
Any special reason for disabling eslint for this line?

Reply:
linter kept complaining because of the any type 😅